server/Common/Utils/JwtUtil.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+
+namespace Common.Utils
+{
+    public class JwtUtil
+    {
+        /// <summary>
+        /// Extracts user id from jwt token
+        /// </summary>
+        /// <param name="token">JWT token generated by IdentityService</param>
+        /// <returns>User identifier</returns>
+        public static Guid GetUserIdFromToken(string token)
+        {
+            token = removeBearer(token);
+
+            var readToken = new JwtSecurityTokenHandler().ReadJwtToken(token);
+            var userId = ((List<Claim>) readToken.Claims).Find(c => c.Type.Equals("sub"));
+            return Guid.Parse(userId.Value);
+        }
+
+        private static string removeBearer(string token)
+        {
+            if (token.Contains("Bearer ")) token = token.Substring(7);
```

Comment:
Do we have a convention of private method starting with small letter? I'm rather accustomed to a I think a standard convention that method names start with a capital no matter the visibility scope